packages/graphql/src/queries/points.graphql

```diff
@@ -0,0 +1,27 @@
+query GetFeeTransfers($address: String!, $cutoff_timestamp: bigint) {
```

Comment:
Nice! Do you think it makes more sense to name this `GetFeeTransfersAggregate` or do you think that fee transfers will always just be the aggregate?

I guess the question would be if we ever see this query having more than just the aggregate values.

Nice job!

Reply:
I imagine it'll always be an aggregate. Maybe just leave it as is for now and we can modify it in the future, if there is ever a need.